.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: node_js
+node_js:
+  - '8'
+  - '7'
+  - '6'
+  - '4'
+before_script:
+  - yarn build
```

Comment:
Why do we need to build?

Reply:
Don't know. This was a copied one, that's why :smile:. I'll remove it.